components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -780,60 +780,93 @@ public IdentityProvider getIdentityProvider(String federatedIdPName, String tena
     public IdentityProvider[] getAllIdentityProviders(String tenantDomain)
             throws IdentityApplicationManagementException {
 
+        List<IdentityProvider> fedIdpList;
```

Comment:
I know we are currently using this method (in ApplicationManagementServiceImpl.java ) to get metadata related to IDPs, local authenticators, request path authenticators etc.

But is ApplicationManagementServiceImpl.java the right place to get this info?